essentials.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ath:
   useLocalSnapshots: false
-  athRevision: "809fb7629c311ebddc6cada91fc865a950155bb0"
-  athImage: "jenkins/ath:acceptance-test-harness-1.63"
+  athRevision: "4e295015ce6b4d5c7a4617bb99dc2968e2436634"
```

Comment:
Ideally, I wouldn't have bumped this yet, but I suspect we cannot do otherwise to not fail anyway (?)

Reply:
You did it right, but as the runATH step uses the last version, we need the last version of ATH, so latest or pin the commits from both, ath source and docker image (more secure, to avoid more breaks due to a future merge on master for example). And as the pin was to the previous versions (1.63 on both, the image and code), it would end up failing anyway.

Reply:
There is a change in `pipeline-library` that is going to break any ath run that is not the one proposed here. So even if you go back to `https://github.com/jenkinsci/jenkins/commit/83c6b52bd2928279cd26f863395d08d6b1aa49d7#diff-65fcc1dbe20813db7c08ca42058bdf1a` that is not going to work anymore.